Sources/Managers/ApiManager.swift

```diff
@@ -18,7 +18,27 @@ class ApiManager {
         }
     }
 
-    func getUrlRequest(searchTerm: String) throws -> URLRequest? {
+    static var decoder: JSONDecoder {
+        let decoder = JSONDecoder()
+        decoder.keyDecodingStrategy = .convertFromSnakeCase
+        decoder.dateDecodingStrategy = .iso8601
+        return decoder
+    }
```

Comment:
Instead of using static modifier on every method/property, you can consider using singleton for APIManager.
https://medium.com/@nimjea/singleton-class-in-swift-17eef2d01d88


or since you are developing in SwiftUI, you can take advantage of @EnvironmentObjects :)